train.py

```diff
@@ -325,15 +328,15 @@ def main(args):
         parameters.update(user_parameters)
```

Comment:
`parameters.update()` is only used for training. But is also called in `__main__.py` when training is specified to check for seed setting.
I'm going to change the way the arguments are handled next, so I'll make it such that the `parameters` dictionary is passed to `deepquest.train()` rather than it loading it again.

Reply:
OK, thanks.